Android/Android.iml

```diff
@@ -13,7 +13,7 @@
     <content url="file://$MODULE_DIR$">
       <excludeFolder url="file://$MODULE_DIR$/.gradle" />
     </content>
-    <orderEntry type="jdk" jdkName="1.8" jdkType="JavaSDK" />
+    <orderEntry type="inheritedJdk" />
```

Comment:
Why this change?

Reply:
The project is Gradle-based, So Android Studio changes it automatically a/c to to it's version. InheritedJDk is default and runs on all computers. Nothing to do with app functionality.